src/mame/barcrest/mpu4_oki_sampled_sound.cpp

```diff
@@ -0,0 +1,199 @@
+// license:BSD-3-Clause
+// copyright-holders:David Haywood, James Wallace
+
+/*
+
+Universal sampled sound program card PCB 683077
+Sampled sound card, using a PIA and PTM for timing and data handling
+The sound hardware is technically twin channel stereo, but there's no evidence to suggest it was ever connected as such
+I've separated the channels here, to tie back at the game level
+
+*/
+
+#include "emu.h"
+
+#include "mpu4_oki_sampled_sound.h"
+
+DEFINE_DEVICE_TYPE(MPU4_OKI_SAMPLED_SOUND, mpu4_oki_sampled_sound, "mpu4okisnd", "Barcrest Sampled Sound Program Card")
+
+mpu4_oki_sampled_sound::mpu4_oki_sampled_sound(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock)
+	: device_t(mconfig, MPU4_OKI_SAMPLED_SOUND, tag, owner, clock),
+		device_mixer_interface(mconfig, *this),
+		m_cb2_handler(*this),
+		m_ptm_ic3ss(*this, "ptm_ic3ss"),
+		m_pia_ic4ss(*this, "pia_ic4ss"),
+		m_msm6376(*this, "msm6376")
```

Comment:
Please keep the initialiser list lined up, and put the colon and commas at the beginning or ends of lines consistently, not a mixture.

Like this:
```
d::d(...) :
    m_x(...),
    m_y(...),
    m_z(...)
{
```

Or this:
```
d::d(...)
    : m_x(...)
    , m_y(...)
    , m_z(...)
{
```